.pre-commit-config.yaml

```diff
@@ -40,3 +42,10 @@ repos:
         language: system
         entry: bazel run //swift/codegen
         pass_filenames: false
+
+      - id: swift-codegen-unit-tests
```

Comment:
Does it mean the codegen tests will run on every commit? It might convenient, but I'm afraid the overhead is way too high to enable it by default.